src/components/Hello.tsx

```diff
@@ -0,0 +1,9 @@
+const Hello: React.FC = () => {
+  return (
+    <div>
+      <h1>Hello, world</h1>
+    </div>
+  );
+};
+
+export default Hello;
```

Comment:
A few words about the file structure. In React projects we usually create a separate folder for each component. Most of the components will have some CSS in future, and it is convenient to keep it in the same folder. Sth like below:

```
components
+-- Counter
|      index.tsx
|      style.scss
+-- Search
|  |      index.tsx
|  +-- SearchButton
|  |      index.tsx
|  |      interfaces.ts
...
```

PS. Sorry, I do not know the real notation system to describe files 😅

Reply:
That was helpful, thanks for the tip. I thought about it but couldn't find an appropriate convention.